suite_test.go

```diff
@@ -80,8 +81,17 @@ var testcases = []struct {
 	{File: "tests/tc47.ber", Error: "eoc child not allowed with definite length"},
 	{File: "tests/tc48.ber", Error: "", IndefiniteEncoding: true}, // Error: "Using of more than 7 "unused bits" in BIT STRING with constrictive encoding form"
 	{File: "tests/tc49.ber", Error: ""},
-	{File: "tests/tc50.ber", Error: "length cannot be less than -1"},
-	{File: "tests/tc51.ber", Error: fmt.Sprintf("length %d greater than maximum %d", 206966894640, MaxPacketLengthBytes)},
+	{File: "tests/tc50.ber", Error: is64bit("length cannot be less than -1", "long-form length overflow")},
+	{File: "tests/tc51.ber", Error: is64bit(fmt.Sprintf("length 206966894640 greater than maximum %v", MaxPacketLengthBytes), "long-form length overflow")},
```

Comment:
@johnweldon @liggitt Open to suggestions. The error message is different between `GOARCH=386` and `GOARCH=amd64`.